speculos/mcu/struct.py

```diff
@@ -1,20 +1,15 @@
 from dataclasses import dataclass
 from typing import Any, Dict, NamedTuple, Optional, Tuple
 
-# from speculos.mcu.apdu import ApduServer
-# from speculos.mcu.seproxyhal import SeProxyHal
-# from speculos.mcu.button_tcp import FakeButton
-# from speculos.mcu.finger_tcp import FakeFinger
-# from speculos.mcu.vnc import VNC
-# from speculos.api.api import ApiRunner
+Pixel = Tuple[int, int]
 
 
 @dataclass
 class Model:
     name: str
-    screen_size: Tuple[int, int]
-    box_position: Tuple[int, int]
-    box_size: Tuple[int, int]
+    screen_size: Pixel
+    box_position: Pixel
+    box_size: Pixel
```

Comment:
screen_size and box_size are more about number of pixels (width, height) than a Pixel which is a x and y coordinate. But that works the same.

Reply:
Indeed. I'm changing it as it may be the same type, but is semantically different to a Pixel.